composer.json

```diff
@@ -21,7 +21,8 @@
     },
     "require-dev": {
         "symfony/phpunit-bridge": "^3.4|^4.0",
-        "symfony/process": "^3.4|^4.0"
+        "symfony/process": "^3.4|^4.0",
+        "symfony/serializer": "^3.4|^4.0"
```

Comment:
It is really necessary? we don't do this for the current ones.

Reply:
@yceruto - it was from the beginning of the repo. Correct me if I'm wrong but I think removing this should go to a separate issue.

Reply:
`symfony/serializer` deps has been introduced in this PR, no? I think is not necessary for tests either, as tests don't check dependencies currently:
https://github.com/yceruto/maker-bundle/blob/c402dac16d4744bf0c1d159ec67bdaf0d3fbd619/tests/Command/FunctionalTest.php#L49

Reply:
Your right. Removed from `composer.json`.